src/js/EditOrderPopup.js

```diff
@@ -0,0 +1,52 @@
+class EditOrderPopup {
+    constructor(editOrderPopupElement) {
+        this._ordersController = null;
+        this._lastOrder = null;
+        this._editOrderPopupElement = editOrderPopupElement;
+        this._statusListSelectElement = this._editOrderPopupElement.find("select.status-list");
+        var statusList = OrderStatus.statusList;
+        for (var i = 0; i < statusList.length; i++) {
+            this._statusListSelectElement.append("<option value="
+                +statusList[i]+">"
+                +statusList[i]+
+            "</option>");
+        }
+        this._statusListSelectElement.dropdown({showOnFocus: false});
+        this._editOrderPopupElement.find(".submit").click(this._onEditFormSubmit.bind(this));
+    }
+
+    setOrdersController(ordersController) {
+        this._ordersController = ordersController;
+    }
+
+    showEditOrderForm(info) {
+        this._lastOrder = info;
+        for (var j in statusColorsList) {
```

Comment:
EditOrderPopup.js:24 Uncaught ReferenceError: statusColorsList is not defined
    at EditOrderPopup.showEditOrderForm (EditOrderPopup.js:24)
    at OrdersTable.showEditOrderform (OrdersTable.js:41)
    at OrdersTab.showEditOrderForm (OrdersTab.js:25)
    at UI.showOrderEditForm (UI.js:44)
    at OrdersController.selectEditOrder (OrdersController.js:35)
    at DisplayOrderPopup._onEditOrderButtonClick (DisplayOrderPopup.js:34)
    at HTMLDivElement.dispatch (jquery-2.1.4.js:4435)
    at HTMLDivElement.elemData.handle (jquery-2.1.4.js:4121)